client/scss/components/_dropdown.legacy.scss

```diff
@@ -123,6 +123,29 @@
 
         li {
             border-width: 0 0 1px;
+
```

Comment:
Putting the styles here means they will only work with the dropup variant, not the dropdown variant.

You may be able to find a spot higher up or elsewhere in this stylesheet that works for both.

You can test the dropdown usage in the reporting section of Wagtail.

(1 focus/styles)